build/plugins/tooltips.html

```diff
@@ -180,13 +184,9 @@
         }
 
         // open tooltip
-        function openTooltip(link) {
-            // if setting is on and already-open tooltip can be found, store
-            // original x position of tooltip
-            let prevX;
-            const prevTooltip = document.getElementById('tooltip');
-            if (options.keepHorizontal === 'true' && prevTooltip)
-                prevX = getRectInPage(prevTooltip).left;
+        function openTooltip(link, prevNextClick) {
```

Comment:
The `prevNextClick` parameter in this function seems redundant because it is never used in this function.